lib/ansible/modules/file.py

```diff
@@ -15,7 +15,7 @@
 short_description: Manage files and file properties
 extends_documentation_fragment: [files, action_common_attributes]
 description:
-- Set attributes of files, symlinks or directories.
+- Set attributes of files, symlinks, their targets, or directories.
```

Comment:
Does "their targets" refer to the symlink target here? TBH this seems like an unclear pronoun reference to me.

Reply:
Yes "their targets" refers to targets  of "symlinks".

Reply:
Thanks for clarifying @prasadpatil49 How about `files, directories, or symlinks and their targets`?

Reply:
Yes this looks better